test/helpers/kubectl.go

```diff
@@ -1195,6 +1195,21 @@ func (kub *Kubectl) Logs(namespace string, pod string) *CmdRes {
 		fmt.Sprintf("%s -n %s logs %s", KubectlCmd, namespace, pod))
 }
 
+// LogsWithLabel returns a CmdRes with command output from the
+// execution of `kubectl logs --previous -l <label string> -n <namespace>`.
+func (kub *Kubectl) LogsWithLabel(namespace string, labelStr string) *CmdRes {
```

Comment:
Nit: maybe let's add "previous" to the method name to avoid any possible confusion.